eng/helix/vstest/runtests.sh

```diff
@@ -60,4 +60,11 @@ export DOTNET_CLI_HOME="$HELIX_CORRELATION_PAYLOAD/home"
 
 export helix="true"
 
-$HELIX_CORRELATION_PAYLOAD/sdk/dotnet vstest $1 --logger:trx
+$DOTNET_ROOT/dotnet vstest $1 -lt >discovered.txt
+if grep -q "Exception thrown" discovered.txt; then
+    echo -e "${RED}Exception thrown during test discovery${RESET}".
+    cat discovered.txt
+    exit 1
+fi
+
+$DOTNET_ROOT/dotnet vstest $1 --logger:trx
```

Comment:
Another inconsistency here is the lack of `--logger:console;verbosity=normal`. Is that intentional?

Reply:
@pakrym added these, most likely for windows specific IIS things, should we just add them to be consistent?

Reply:
Or maybe we should switch this to a HelixLogSettings so test projects can customize on demand and feed it into the scripts, not sure this is something that we need more granularity on...

Reply:
Suggest making it consistent. If we have trouble with excess logging, can revisit in the future.

Reply:
I'm making it consistent in https://github.com/aspnet/AspNetCore/pull/8094

Reply:
Ok I'll leave this for now in this PR if you are changing it, I'm just rerunning helix before merging, there was a flaky helix failure